### PR TITLE
Add default values to --help

### LINKED ISSUE #993: Polaris help should show the default values
`synopsysctl create polaris --help` should display the default values

### PR DESCRIPTION
**Pull request by jdartigalongue**
Closes #993

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/998#" title="Author self-approved">jdartigalongue</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.11.x/OWNERS)~~ [jdartigalongue]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/polaris/ctl_polaris.go [hunk 1]
```diff
@@ -132,12 +133,12 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
...
+	cmd.Flags().StringVar(&ctl.UploadServerSize, "uploadserver-size", ctl.UploadServerSize, fmt.Sprintf("PVC size to use for uploadserver. Default: %s", GetPolarisDefault().PolarisDBSpec.UploadServerDetails.Storage.StorageSize))
+	cmd.Flags().StringVar(&ctl.EventstoreSize, "eventstore-size", ctl.EventstoreSize, fmt.Sprintf("PVC size to use for eventstore. Default: %s", GetPolarisDefault().PolarisDBSpec.EventstoreDetails.Storage.StorageSize))
+	cmd.Flags().StringVar(&ctl.MongoDBSize, "mongodb-size", ctl.MongoDBSize, fmt.Sprintf("PVC size to use for mongodb. Default: %s", GetPolarisDefault().PolarisDBSpec.MongoDBDetails.Storage.StorageSize))
+	cmd.Flags().StringVar(&ctl.DownloadServerSize, "downloadserver-size", ctl.DownloadServerSize, fmt.Sprintf("PVC size to use for downloadserver. Default: %s", GetPolarisDefault().PolarisSpec.DownloadServerDetails.Storage.StorageSize))
+	cmd.Flags().StringVar(&ctl.ReportStorageSize, "reportstorage-size", ctl.ReportStorageSize, fmt.Sprintf("PVC size to use for reportstorage. Only applicable only if --enable-reporting is set to true. Default: %s", GetPolarisDefault().ReportingSpec.ReportStorageDetails.Storage.StorageSize))
```
Comment (msenmurugan) [inline pkg/polaris/ctl_polaris.go]: can you pls remove one of the only?

Comment (jdartigalongue) [reply pkg/polaris/ctl_polaris.go]: Done

File: pkg/polaris/ctl_polaris.go [hunk 2]
```diff
@@ -118,9 +119,9 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.EnvironmentDNS, "environment-dns", ctl.EnvironmentDNS, "Environment DNS")
 	cmd.Flags().StringVar(&ctl.ImagePullSecrets, "pull-secret", ctl.ImagePullSecrets, "Pull secret")
 	cmd.Flags().StringVar(&ctl.StorageClass, "storage-class", ctl.StorageClass, "Set the storage class to use for all the PVC")
-	cmd.Flags().BoolVar(&ctl.EnableReporting, "enable-reporting", false, "Send this flag if you wish to enable ReportingPlatform")
+	cmd.Flags().BoolVar(&ctl.EnableReporting, "enable-reporting", false, fmt.Sprintf("Send this flag if you wish to enable ReportingPlatform. Default: %s", strconv.FormatBool(GetPolarisDefault().EnableReporting)))
 	cmd.Flags().StringVar(&ctl.GCPServiceAccount, "gcp-service-account-path", ctl.GCPServiceAccount, "Google Cloud Service account")
-	cmd.Flags().StringVar(&ctl.IngressClass, "ingress-class", ctl.IngressClass, "The name of the ingress class to use. Default is nginx.")
+	cmd.Flags().StringVar(&ctl.IngressClass, "ingress-class", ctl.IngressClass, fmt.Sprintf("The name of the ingress class to use. Default: %s", GetPolarisDefault().IngressClass))
```
Comment (yashbhutwala) [inline pkg/polaris/ctl_polaris.go]: All the string var, in cobra, the third parameter is already meant for `default` values, so you could just put it there, and change the fourth parameter to be string as is.

Comment (yashbhutwala) [reply pkg/polaris/ctl_polaris.go]: This, for example https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.11.x/pkg/blackduck/ctl_blackduck.go#L162

Comment (jdartigalongue) [reply pkg/polaris/ctl_polaris.go]: Updated

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/998#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/998#pullrequestreview-303499557" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.11.x/OWNERS)~~ [jdartigalongue,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**